qa/tasks/cephfs/test_client_limits.py

```diff
@@ -272,20 +272,17 @@ def test_client_blocklisted_oldest_tid(self):
         # Wait for the health warnings. Assume mds can handle 10 request per second at least
         self.wait_for_health("MDS_CLIENT_OLDEST_TID", max_requests // 10, check_in_detail=str(self.mount_a.client_id))
 
-        # why a multiplier of 20, you may ask - I arrieved at this from some debugs
-        # that I put when testing the fix in a vstart cluster where its a ratio of
-        # encoded session information to the number of completed requests.
-        self.config_set('mds', 'mds_session_metadata_threshold', max_requests*20);
+        self.config_set('mds', 'mds_session_metadata_threshold', 5000);
```

Comment:
can we still have this expressed as a quantity derived from the test inputs or config?

Reply:
The idea is to have a lower value so that the probability of hitting with a higher request count is high. The config thing is a bit of a overkill - qa tests are dumped with lots of hardcoded values which are a bit off-putting, but, if they are supplemented by code commentary then its pretty much accepted.

Reply:
In this case, the commentary was removed. However, I personally prefer a code that tells the story on its own by the means of variable and constant naming, and deriving other values from a small number of initial randomized input.

By config, I meant a couple of constants that you use with multipliers and relations throughout the same test. This makes it easier change the "config" without invalidating the test invariants.

this 5000 is clearly related to the `max_requests` above, so I don't see a reason to hardcode it, especially in a place that's not adjacent to the max_request initialization for easy validation.

Reply:
> In this case, the commentary was removed.

Yeh, I'm adding some explanation for it.

> However, I personally prefer a code that tells the story on its own by the means of variable and constant naming, and deriving other values from a small number of initial randomized input.

I would consider it as a separate effort which would cleanup the hardcoded values we have. +1 for this.

> 
> By config, I meant a couple of constants that you use with multipliers and relations throughout the same test. This makes it easier change the "config" without invalidating the test invariants.
> 
> this 5000 is clearly related to the `max_requests` above, so I don't see a reason to hardcode it, especially in a place that's not adjacent to the max_request initialization for easy validation.

Not really - You _could_ hit the threshold with max_requests, but I deliberately kept it higher to avoid any edge-cases.

Let's create a tracker for the cleanup.